grouper/fe/templates/help.html

```diff
@@ -17,8 +18,95 @@
+<p>Users are automatically populated into Grouper as they visit the web UI.  We're working
+on a manual way to pre-populate users (see user_admin below).</p>
+
+<h3>Who can create permissions?</h3>
+<p>A user with the permission ({{permission(create_perm)}}, prefix.*) can create any
+permission starting with "prefix." We strongly encourage namespacing using the character
+'.' as a separator.</p>
+
+<h3>Who can grant permissions?</h3>
+<p>Generally, a user with the permission ({{permission(grant_perm)}}, prefix.*) can grant
+any permission starting with "prefix." with any argument.  As a special case, the
+permission ({{grant_perm.name}}, {{grant_perm.name}}/prefix.*) allows a user to
+grant the permission to grant ({{grant_perm.name}}, prefix.*).</p>
+
+<h3>How does auditing work?</h3>
+<!-- FIXME(mildorf): does the argument matter for grouper.auditor.permission? -->
+<p>Grouper permissions have a bit that defines whether they are audited or not.  Groups
+with at least one audited permission become audited groups.  Users with the
+{{permission(audit_perm)}} permission (any argument) are considered auditors.  Only
+auditors can be the manager or owner of an audited group.</p>
+
+<!-- TODO: change once auditing tooling is ready! -->
+<p>We're working on tooling to facilitate regular monitoring of both members of audited
+groups and groups with audited permissions.</p>
+
+<h3>How do I set up Grouper?</h3>
+<p>See the README that comes with Grouper. Basically, after a pip install, you need to
+configure grouper to find a SQL-style backing database and stand up processes to serve the
+read-write web UI and read-only programmatic API.</p>
+
+<h3>How do I set up the first groups and permissions?</h3>
+<p>There are three administrative flags, corresponding to full authority to administer
+  groups, users, and permissions. These flags can be set for any account and grant powers
+  independent of the usual graph-based way and are given out manually via the following
+  commands:</p>
+
+<ul><li>bin/grouper-ctl -c config/dev.yaml -vv capabilities add user@example.com <strong>group_admin</strong></li>
+<li>bin/grouper-ctl -c config/dev.yaml -vv capabilities add user@example.com <strong>user_admin</strong></li>
+<li>bin/grouper-ctl -c config/dev.yaml -vv capabilities add user@example.com <strong>permission_admin</strong></li></ul>
+
+<p>A <strong>group_admin</strong> has the power to manipulate the graph structure at will.
+For example, they can create any group, add any user to any group as any role, provided
+that auditing requirements aren't violated.  A <strong>user_admin</strong> has the power
+to disable/enable a user account but no special power with respect to group
+membership. Finally, a <strong>permission_admin</strong> has the power to create any
+permission.</p>
+
+<h3>What if my questions aren't answered here?</h3>
+<p>This list isn't comprehensive. Please contact the admins of this site ({{
+  current_admins }}) for help and
```

Comment:
No, it shouldn't contain HTML.  Is there a way to ensure that a string embeds as literal text?  The thought was that different organizations might have different ideas about reaching out for help, not necessarily by email, so that less is more in this case.  I'll change it to something more descriptive like "how_to_get_help" with the intent to configure it as something like "email user1@, user2@, and team@, or reach out on #chat."